cli/cmd/lib_manager.go

```diff
@@ -240,6 +240,13 @@ func runManagerUpdateCommand(entrypoint string, clusterConfig *clusterconfig.Con
 			"CORTEX_TELEMETRY_SEGMENT_WRITE_KEY=" + os.Getenv("CORTEX_TELEMETRY_SEGMENT_WRITE_KEY"),
 			"CORTEX_CLUSTER_CONFIG_FILE=" + mountedConfigPath,
 			"CORTEX_CLUSTER_WORKSPACE=" + clusterWorkspace,
+			"CORTEX_IMAGE_PYTHON_SERVE=" + consts.DefaultImagePythonServe,
```

Comment:
Apparently, yeah. I would actually suggest keeping it as it is, because:
1. It's consistent with how all other images are referenced in the other manifests.
1. There's only one place where the default paths are defined: in `consts` package.

Reply:
Makes sense! I'll resolve this comment now